src/Illuminate/Auth/Access/Gate.php

```diff
@@ -162,7 +164,7 @@ public function check($ability, $arguments = [])
      *
      * @param  string  $ability
      * @param  array  $arguments
-     * @return callable
+     * @return mixed
```

Comment:
`return [$this->resolvePolicy($this->policies[get_class($arguments[0])]), $ability];` is an array